docs/src/pages/components/typography/typography.md

```diff
@@ -59,7 +59,7 @@ Hopefully, you might be able to take advantage of the [`typography`](/customizat
 The Typography component uses the `variantMapping` property to associate a UI variant with a semantic element.
 It’s important to realize that the style of a typography is independent from the semantic underlying element.
 
-- You can change the underlying element for a one time occassion with the `component` property:
+- You can change the underlying element for a one time occasion with the `component` property:
```

Comment:
As we are going to invalidate the crowding translation, we might have well fix this:
```suggestion
- You can change the underlying element for a one time occasion with the `component` prop:
```